crypto/test/tests.go

```diff
@@ -32,3 +32,48 @@ func RandomSignerRFC6979(tb testing.TB) user.Signer {
 
 	return user.NewAutoIDSignerRFC6979(p.PrivateKey)
 }
+
+// SignedComponent describes component which can signed and the signature may be verified.
+type SignedComponent interface {
+	SignedData() []byte
+	Sign(neofscrypto.Signer) error
```

Comment:
Likely some generic magic can be used here, but two interfaces are kinda OK for test as well.